Book-A-Desk.Domain/QueriesHandler.Offices.fs

```diff
@@ -40,4 +40,17 @@ module rec OfficeQueriesHandler =
                         AvailableDesks = office.BookableDesksPerDay - bookingsForOffice.Length
                     }
                 return officeAvailability
-        }
\ No newline at end of file
+        }
+    
+    let getOfficeName (officeReference: string) getOffices =
```

Comment:
So, the Domain shouldn't be the spot where you transform infrastructure types (such as this office) to an Office Id. You probably should only pass the (officeId : OfficeId) instead and do the parsing in the API project.

Also, make sure you handle the Guid.Parse failing